src/features/events/components/EventWarningIcons.tsx

```diff
@@ -73,8 +73,7 @@ const EventWarningIconsSansModel: FC<{
     if (!hasContact) {
       icons[0] = (
         <WarningSlot
-          key="contact"
-          icon={<FaceRetouchingOff color="error" />}
+          icon={<FaceRetouchingOff key="contact" color="error" />}
```

Comment:
What is the purpose of this change (and the others like it)? Will this not cause warnings because the `WarningSlot` elements now lack a `key`?

Reply:
I did it in response to a console warning that elements were missing their `key` property, changing it to this made the key end up on the right element in the dom and the warning went away...